src/routes/components/info.svelte

```diff
@@ -0,0 +1,44 @@
+<script>
+	import { fly, fade } from 'svelte/transition';
+	let visible = 'beginning';
+	function handleClick() {
+		switch (visible) {
+			case 'beginning':
+				visible = 'stack';
+				break;
+			case 'stack':
+				visible = 'next';
+				break;
+			default:
+				visible = 'beginning';
+				break;
+		}
+	}
```

Comment:
You could rewrite the `switch` statement using an object for mapping values declaratively:

```suggestion
   const currentToNextVisible = {
     beginning: 'stack',
     stack: 'next',
   };

	function handleClick() {
     visible = currentToNextVisible[visible] ?? 'beginning';
	}
```

Reply:
Love it!